drafts/bitcartel-payment-disclosure/draft.rst

```diff
@@ -0,0 +1,339 @@
+::
+
+  ZIP: 303
+  Title: Payment Disclosure
+  Owners: Deirdre Connolly <deirdre@zfnd.org>
+  Original-Author: Simon Liu
+  Status: Implemented (zcashd)
+  Category: Informational
+  Created: 2017-02-22
+  License: MIT
+
+
+Abstract
+--------
+
+This ZIP describes a method of proving that a payment was sent to a shielded address.  In the typical case, this means enabling a sender to present a proof that they transferred funds to a recipient's shielded address.  The method described will be compatible with the Zcash Protocol [#protocol]_ deployed at the launch of the Zcash network. See zcash issues `2036 <https://github.com/zcash/zcash/issues/2036>`_ and `737 <https://github.com/zcash/zcash/issues/737>`_ for context.
+
+
+Motivation
+----------
+Payment disclosure is useful in a number of situations, for example:
+
+* A sender may need to prove that their payment was sent and received by a recipient.  For example, a customer paid too much for an item and would like to claim a refund from the vendor.
+* A third party needs to verify that a payment between sender and recipient was executed successfully.  For example, a regulator needs to confirm a transfer of funds between two parties.
```

Comment:
Note that the _recipient_ of a shielded transaction may wish to share visibility of a shielded transaction with a third party. The ZIP should specify whether a payment disclosure can be created by the recipient and, if so, how.

Reply:
I think the recipient could only create a disclosure in the `transparent --> shielded` case. In other cases, they don't have the information required for the proof. However, I wonder about the motivation for a recipient-disclosed payment.